CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 3.0.0)
 project(cmake_wrapper)
 
-include(conanbuildinfo.cmake)
+include("${CMAKE_BINARY_DIR}/conanbuildinfo.cmake")
 conan_basic_setup()
 
-add_subdirectory("source_subfolder")
```

Comment:
This is common across all recipes, but don't rename this when it isn't for a very good reason

Reply:
ok